packages/flutter/lib/src/material/no_splash.dart

```diff
@@ -64,4 +65,16 @@ class NoSplash extends InteractiveInkFeature {
   @override
   void paintFeature(Canvas canvas, Matrix4 transform) {
   }
+
+  @override
+  void confirm() {
```

Comment:
@polina-c - does this look right to you?

Reply:
yes! thank you!